lib/action_client/log_subscriber.rb

```diff
@@ -0,0 +1,19 @@
+module ActionClient
+  class LogSubscriber < ActiveSupport::LogSubscriber
+    def submit(event)
+      client = event.payload[:client]
+      action_name = event.payload[:action_name]
+      request = event.payload[:request]
+      action_name = color("#{client.class}##{action_name}", CYAN, true)
+      http = color("#{request.method} #{request.url}", CYAN, true)
+
+      info "#{action_name} - #{http} - #{duration(event)}"
```

Comment:
A hyphen and parens seems a bit much. How about dropping the second hyphen here?